microsoft/testsuites/mshv/mshv_install.py

```diff
@@ -0,0 +1,97 @@
+
+from lisa import Logger, Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.testsuite import TestResult
+from lisa.tools import Cp, Ls, Reboot
+from lisa.util import SkippedException
+
+
+@TestSuiteMetadata(
+    area="mshv",
+    category="functional",
+    description="""
+    This test suite is to test VM working well after updating MSHV on VM
+    and rebooting.
+    """,
+)
+class MshvHostInstallSuite(TestSuite):
+    CONFIG_BINPATH = "mshv_binpath"
+
+    _test_path_init_hvix = (
+        Path("/home/cloud") / "hvix64.exe"
+    )
+
+    _init_path_init_kdstub = (
+        Path("/home/cloud") / "kdstub.dll"
+    )
+
+    _init_path_init_lxhvloader = (
+        Path("/home/cloud") / "lxhvloader.dll"
+    )
+
+    _test_path_dst_hvix = (
+        Path("/boot/efi/Windows/System32") / "hvix64.exe"
+    )
+
+    _test_path_dst_kdstub = (
+        Path("/boot/efi/Windows/System32") / "kdstub.dll"
+    )
+
+    _test_path_dst_lxhvloader = (
+        Path("/boot/efi") / "lxhvloader.dll"
+    )
+
+    @TestCaseMetadata(
+        description="""
+        This test case will
+        1. Update to new MSHV components over old ones in a
+            pre-configured MSHV image
+        2. Reboot VM, check that mshv comes up
+
+        The test expects the directory containing MSHV binaries to be passed in
+        the mshv_binpath variable.
+        """,
+        timeout=120,
+    )
+    def verify_mshv_install_succeeds(
+        self,
+        log: Logger,
+        node: Node,
+        variables: Dict[str, Any],
+        log_path: Path,
+        result: TestResult,
+    ) -> None:
+        binpath = variables.get(self.CONFIG_BINPATH, "")
+        if not binpath:
+            raise SkippedException(
+                "Requires a path to MSHV binaries to be passed via mshv_binpath"
+            )
+
+        test_hvix_file_path = Path(binpath) / "hvix64.exe"
+        test_kdstub_file_path = Path(binpath) / "kdstub.dll"
+        test_lxhvloader_file_path = Path(binpath) / "lxhvloader.dll"
+
+        log.info(f"binpath: {binpath}")
+
+        # Copy Hvix64.exe, kdstub.dll, lxhvloader.dll into test machine
+        node.shell.copy(test_hvix_file_path, self._test_path_init_hvix)
+        node.tools[Cp].copy(self._test_path_init_hvix,
+                            self._test_path_dst_hvix, sudo=True)
+
+        node.shell.copy(test_kdstub_file_path, self._init_path_init_kdstub)
+        node.tools[Cp].copy(self._init_path_init_kdstub,
+                            self._test_path_dst_kdstub, sudo=True)
+
+        node.shell.copy(test_lxhvloader_file_path, self._init_path_init_lxhvloader)
+        node.tools[Cp].copy(self._init_path_init_lxhvloader,
+                            self._test_path_dst_lxhvloader, sudo=True)
+
+        reboot_tool = node.tools[Reboot]
+        reboot_tool.reboot_and_check_panic(log_path)
+
+        # 2. check that mshv comes up
+        mshv = node.tools[Ls].path_exists("/dev/mshv", sudo=True)
+        assert mshv
```

Comment:
Add message to the assertion. So once it's failed, the error message can indicate what happened, and how to deal with it.